### DISCUSSION TITLE
Why is hcaptcha and mapbox required components?

### DISCUSSION THREAD

**Discussion by TheGorf** [mawoka-myblock/ClassQuiz#341] [General]
I'd like the option for self-hosted to remove hcaptcha and mapbox. Is that something we can adapt easily? hCaptcha isn't necessary for how I'm using ClassQuiz. And I'm wondering if someone could explain what the point of mapbox is? I really don't want any type of geo tagging or anything like that. It won't do any good any way since we are running this all in an RFC1918 address space.

**Comment by mawoka-myblock** [comment]
They aren't required. Just leave them "as is" and nothing is loaded. The feature where Mapbox has been used, isn't available anymore. hCaptcha can just not be enabled (when starting a game) and no proprietary code from these services is ever loaded.

**Reply by TheGorf** [reply]
Ok makes sense. Might be a good item to update on the documentation since the [self-hosted page](https://classquiz.de/docs/self-host) states that they are are a **required** pre-requisite.

**Reply by mawoka-myblock** [reply]
Is now fixed.
